src/main/java/Duke.java

```diff
@@ -1,10 +1,78 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static Task [] taskList = new Task[100];
```

Comment:
Will it be better to give the constant 100 a constant name like MAX_TASK_NUMBER?

Reply:
really useful tip, thank you!